test/resources/dgl_gcn/gcn.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""GCN using DGL nn package
+References:
+- Semi-Supervised Classification with Graph Convolutional Networks
+- Paper: https://arxiv.org/abs/1609.02907
+- Code: https://github.com/tkipf/gcn
+"""
+import mxnet as mx
+from mxnet import gluon
+import dgl
```

Comment:
do we ever explicitly use dgl by itself? can prob remove thisimport statements

also, can we reorganize import statements to be PEP8 compliant? see https://www.python.org/dev/peps/pep-0008/#imports

"Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports."

Reply:
Yes, we should import it explicitly. It is a standalone lib.
Same as mxnet.

Reply:
Can we still reorganize the import statements? Also, it doesn't look like we are using "dgl" by itself anywhere in the script as you are importing modules from dgl by name, so we can prob remove this import statement.

Reply:
oh oops, just saw you made the change. thanks

Reply:
I see. import dgl is not needed